examples/scalajs-react/.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea
+target
+
```

Comment:
* [ ] Build up [a decent `.gitignore`](https://github.com/tastejs/todomvc/blob/master/examples/angularjs-perf/.gitignore).
* [ ] The `.idea` should be in your global gitignore.
* [ ] Remove the empty line at the end ;)